scripts/build-mimalloc-debug.sh

```diff
@@ -27,3 +27,4 @@ cmake "${CMAKE_FLAGS[@]}" . \
 ninja
 
 cp libmimalloc-debug.a $BUN_DEPS_OUT_DIR/libmimalloc-debug.a
+cp CMakeFiles/mimalloc-obj.dir/src/static.c.o $BUN_DEPS_OUT_DIR/libmimalloc-debug.o
```

Comment:
Why picking `.o` directly in the CMakeFiles intermediate files? Their locations and names is likely to change.

Reply:
(see other comment)

Reply:
> (see other comment)

It would be better if the mimalloc build artefacts is a `.o` file, then ar is  used to create the matching `.a`. The current behaviour is enforced by the mimalloc buildsystem behaviour at outputting only archive file and a need to use to `.o` file for an undocumented reason.